### DISCUSSION TITLE
blog/todo-app-in-angular-nestjs-ngrx/

### DISCUSSION THREAD

**Discussion by giscus** [msindev/penguincoders.net#6] [Blog Comments]
# blog/todo-app-in-angular-nestjs-ngrx/

In this post, we will be creating a fully functional ToDo application in a Nx monorepo, using Angular for Frontend and NestJs for Backend. The ToDo app will also provide Angular’s state management using NgRx. Our frontend app will be using Tailwind to style our components.

https://penguincoders.net/blog/todo-app-in-angular-nestjs-ngrx/

**Comment by rajrohit98** [comment]
Good Work Mohit Singh.

Really helpful.
